rtt/base/TaskCore.hpp

```diff
@@ -337,6 +337,17 @@ namespace RTT
             return ee;
         }
 
+        /**
+         * The name of this TaskCore
+         *
+         * Used for debugging and/or tracing purposes
+         */
+        std::string getName() const;
```

Comment:
Return type `const std::string &` like in [const std::string& TaskContext::getName() const](https://github.com/orocos-toolchain/rtt/blob/8e995a22ab8a545748fae84dc7095a3007edfee0/rtt/TaskContext.hpp#L128)?